src/apiCalls.js

```diff
@@ -0,0 +1,14 @@
+function fetchData(url) {
+  return fetch(url)
+  .then(response => {
+    if (response.ok) {
+      return response.json()
+    } else if (response.status >= 500) {
+      throw new Error('The server\'s messed up!')
+    } else if (response.status >= 400 && response.status < 500) {
+      throw new Error('This is a level 400 error')
```

Comment:
Glad to see you thinking of those errors and what you'd wanna communicate to the user